src/python/base/utils.py

```diff
@@ -948,21 +947,3 @@ def cpu_count():
 
   return environment.get_value('CPU_COUNT_OVERRIDE',
                                multiprocessing.cpu_count())
-
-
-# TODO(mbarbella): Delete this once fully migrated to Python 3.
-def newstr_to_native_str(s):
-  try:
-    newstr = str(s, 'utf-8')
-  except TypeError:
-    # Python 3 does not support an encoding argument when given a unicode str.
-    newstr = str(s)
-
-  return future_utils.native(newstr).encode('utf-8')
-
-
-def exc_clear():
```

Comment:
This is not removed from here.

```
src/python/bot/startup/heartbeat.py:        utils.exc_clear()
src/python/bot/startup/run_bot.py:    utils.exc_clear()
src/python/bot/tasks/commands.py:  utils.exc_clear()
```

Reply:
I have removed it from all required places.